javatests/arcs/showcase/references/Arcs.kt

```diff
@@ -1,176 +0,0 @@
-@file:Suppress("EXPERIMENTAL_IS_NOT_ENABLED")
-
-package arcs.showcase.references
-
-import android.content.Context
-import androidx.lifecycle.LifecycleOwner
-import androidx.lifecycle.LifecycleRegistry
-import arcs.core.allocator.Allocator
-import arcs.core.allocator.Arc
-import arcs.core.common.ArcId
-import arcs.core.common.toArcId
-import arcs.core.data.Plan
-import arcs.core.host.AbstractArcHost
-import arcs.core.host.EntityHandleManager
-import arcs.core.host.ParticleState
-import arcs.core.host.toRegistration
-import arcs.core.host.SchedulerProvider
-import arcs.core.storage.ActivationFactory
-import arcs.jvm.host.ExplicitHostRegistry
-import arcs.jvm.host.JvmSchedulerProvider
-import arcs.jvm.util.JvmTime
-import arcs.sdk.Particle
-import arcs.sdk.android.storage.ServiceStoreFactory
-import arcs.sdk.android.storage.service.ConnectionFactory
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.runBlocking
-import kotlinx.coroutines.withTimeout
-import kotlin.coroutines.EmptyCoroutineContext
-
-/** Container for WriteRecipe specific things */
-@ExperimentalCoroutinesApi
-class ArcsStorage(private val arcs: Arcs) {
```

Comment:
I would like to keep this DAO-like object wrapper if possible...

Reply:
Reverted!